### PR TITLE
submitEditRequest method added. test added.

### PR DESCRIPTION
**Pull request by ahodge10**
Submit edit request functionality complete. Testing added in testing class.

### CODE REVIEW

File: src/main/java/com/revature/services/ProjectService.java [hunk 1]
```diff
@@ -331,5 +331,21 @@ public Project findById(String id) {
...
+	
+    /**
+     * 
+     * @param project
+     * @return
+     */
+    public boolean submitEditRequest(Project project) {
+        if(project != null) {
+            System.out.println(project.getStatus());
+//            System.out.println(project.getOldProject().getBatch());
```
Comment (jurczakn) [inline src/main/java/com/revature/services/ProjectService.java]: Remove commented out code

**Review by jurczakn — CHANGES_REQUESTED** [review]

**Comment by ahodge10** [comment]
Requested changes completed.

File: src/main/java/com/revature/security/CustomAuthenticationFilter.java [hunk 2]
```diff
@@ -1,134 +1,134 @@
...
+			 */
+			e.printStackTrace();
+			String ipAddress = ((HttpServletRequest) request).getHeader("X-FORWARDED-FOR");
+			if (ipAddress == null) {
+				ipAddress = request.getRemoteAddr();
+			}
+
+			String appUrl = request.getScheme() + "://" + request.getLocalAddr();
+			System.out.println("URL     " + appUrl);
+			// throw new RuntimeException(ipAddress + " " + appUrl, e);
```
Comment (jurczakn) [inline src/main/java/com/revature/security/CustomAuthenticationFilter.java]: still has commented out code

File: src/main/java/com/revature/security/CustomAuthenticationFilter.java [hunk 3]
```diff
@@ -1,134 +1,134 @@
...
+			 * including the IP address of the requester.
+			 */
+			e.printStackTrace();
+			String ipAddress = ((HttpServletRequest) request).getHeader("X-FORWARDED-FOR");
+			if (ipAddress == null) {
+				ipAddress = request.getRemoteAddr();
+			}
+
+			String appUrl = request.getScheme() + "://" + request.getLocalAddr();
+			System.out.println("URL     " + appUrl);
```
Comment (jurczakn) [inline src/main/java/com/revature/security/CustomAuthenticationFilter.java]: still has sys out

**Review by jurczakn — CHANGES_REQUESTED** [review]

**Comment by ahodge10** [comment]
CustomAuthenticationFilter and tests updated to pass Jenkins build

**Review by jurczakn — APPROVED** [review]
ok

**Pull request merged**